docs/source/llms/tracing/index.rst

```diff
@@ -690,3 +690,85 @@ Client API
         request_id=parent_span.request_id,
         attributes={"attribute3": "value3", "attribute4": "value4"},
     )
+
+Q: How can I see the stack trace of a Span that captured an Exception?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The MLflow UI does not display Exception types, messages, or stacktraces if faults occur while logging a trace. 
+However, the trace does contain this critical debugging information as part of the Span objects that comprise the Trace. 
+
+The simplest way to retrieve a particular stack trace information from a span that endured an exception is to retrieve the trace directly in 
+an interactive environment (such as a Jupyter Notebook). 
+
+Here is an example of intentionally throwing an Exception while a trace is being collected and a simple way to view the exception details:
+
+.. code-block:: python
+
+    import mlflow
+
+    experiment = mlflow.set_experiment("Intentional Exception")
+
+    with mlflow.start_span(name="A Problematic Span") as span:
+        span.set_inputs({"input": "Exception should log as event"})
+        span.set_attribute("a", "b")
+        raise Exception("Intentionally throwing!")
+        span.set_outputs({"This": "should not be recorded"})
+
+When running this, an Exception will be thrown, as expected. However, a trace is still logged to the active experiment and can be retrieved as follows:
+
+Note that the ``get_trace`` API is **only available** for retrieving traces from the current active session. If you are attempting to retrieve 
+a trace from a previous session, you must use the ``search_traces`` client API (shown below).
+
+
+.. code-block:: python
+
+    trace = mlflow.get_trace(span.request_id)
+    trace_data_spans = trace.data.spans
+    trace_data.to_dict()
+
+In an interactive environment, such as a Jupyter Notebook, ``stdout`` will render an output like this:
```

Comment:
```suggestion
    print(trace_data.to_dict())

Output:
```

can we use `print`?

Reply:
print collapses the output into a string-wrapped blob that is impossible to read, whereas the Jupyter renderer will pretty print dictionaries. 
I'm fine with adding the print, I just left it out to make it more visually appealing for users who copy the code to test it out

Reply:
![Screenshot 2024-06-10 at 10 07 13 PM](https://github.com/mlflow/mlflow/assets/39283302/51e883b4-f299-4c3a-8e53-2e2361a9d41a)
(comparison)

Reply:
@BenWilson2 

can we use `trace_data.to_json(pretty=True)`?

Reply:
TraceData doesn't have a `to_json` method

Reply:
Got it, then I'm fine with the current code.

Reply:
I actually just changed it to pprint. I think it's pretty close to the `Output` block in Jupyter, but now people running in command line can use it :)